samples/subsys/logging/kernel_event_logger/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _kerneleventlogger_sample:
+
+Kerneleventlogger Sample
+######################
+
+Overview
+********
+
+A simple application that demonstrates use of kernel event
+logger feature.Two threads (A and B) of same priority are
+created and another thread with less priority (i.e event_
```

Comment:
Words that end with an underscore are meaningful to the doc gen process, so break the line before ``event_`` so the line doesn't end with an underscore. Also some clarity edits:

     created and a third thread with lower priority (i.e.,
     event_logger_thread) is also created, which reads
     the events from